docs/networks/l16-testnet.md

```diff
@@ -51,6 +51,16 @@ curl https://raw.githubusercontent.com/lukso-network/network-configs/l16-dev/l16
 
 This will download `docker-compose.yaml`, `Makefile`, `configs` and `.env` files.
 
+### Change node name (Optional)
+1. open docker file using any text editor. for example vim
+```bash
+vim .env
+```
+2. Change `NODE_NAME` to your expected string
+
+# change 
+```
```

Comment:
is this expected?

<img width="643" alt="image" src="https://user-images.githubusercontent.com/477945/158546414-61afec5e-bce8-4cf3-a86c-dadfaf3a9ab5.png">